jkamz/Questioner
# #163371828 add app configs

### What does this PR do?
This PR adds app configs for v2

### Description of the tasks to be completed
The tasks for this PR include:

- Add app dependencies to requirements.txt
- Add app and environment configurations
- Add create function and add run app functionality

### How can this be manually tested?
Clone this repository on local machine and CD into project folder

`$ git clone https://github.com/jkamz/Questioner`

`$ cd Questioner`

Check out to the PR's branch
`$ git checkout ch-app-config-163371828`

Activate virtual environment and export environment variables

run `flask run`

### What are the relevant Pivotal Tracker stories?
#163371828

##Screenshot

![runflask](https://user-images.githubusercontent.com/11965186/51495634-1ee05e80-1dce-11e9-9b17-e90ed4055307.png)
